sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -500,8 +513,6 @@ object ScalaJSPluginInternal {
       relativeSourceMaps   := false,
       persistLauncher      := false,
 
-      skip in packageJSDependencies := true,
```

Comment:
No, it's the default: https://github.com/sbt/sbt/blob/v0.13.8/main/src/main/scala/sbt/Defaults.scala#L140